pip/qsharp/__init__.py

```diff
@@ -27,6 +29,8 @@
     "init",
     "eval",
     "run",
+    "set_quantum_seed",
+    "set_classical_seed",
```

Comment:
I have no suggestions on how to improve this, but I feel like it's really unclear whether these apply to any/all of `qsharp.eval`, `qsharp.run` or `qsharp.compile` (QIR generation). How can we make it clearer to the user exactly what they're changing?

Reply:
(Also, qsharp.estimate - quantum seed would not be relevant but classical seed would impact the results)

Reply:
All great points, and the answer is that the setting is respected all the scenarios except debugging. But as you point out, neither `compile` for QIR generation nor `estimate` for resource estimation do any quantum simulation, so even though the quantum setting is passed through in those cases only the classical setting changes behavior.

Reply:
To put it another way, since debugging can't be invoked through Python and these are only exposed through Python, once you make the call `qsharp.set_classical_seed(val)` it be respected for all calls to `qsharp` after that until you either reinitialize or set the seed again. The same is true for `set_quantum_seed` with the caveat that it doesn't affect those calls that don't perform quantum simulation, namely `compile` and `estimate`.

Reply:
I can wrap my head around it now, thanks - but maybe it's worth explaining in the doc comments.